### PR TITLE
Introduce an option to configure minimum coverage percantages

### PR DESCRIPTION
**Pull request by kl4n4**
This might be helpful to others as well since in our company we do not have a 100% coverage requirement. So we introduced the option to reconfigure these limits (the default still remains at 100).

Besides that I also updated the `release.yml` to match that from the `tangro-actions-template`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,14 +59,30 @@ steps:
...
+There is also an option to adapt the minimum coverage percentage limits that need to be achived for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
+
+## Example with alternate coverage minimums
+
+```yml
+steps:
+  - name: Collect Coverage
+    uses: tangro/actions-coverage@1.1.0
+    env:
+      COVERAGE: 94
```
Comment (ankri) [inline README.md]: I would have preferred for the configuration values to be [parameters](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith) but am fine with using environment variables.

File: README.md [hunk 2]
```diff
@@ -59,14 +59,30 @@ steps:
...
-    uses: tangro/actions-coverage@1.0.0
+    uses: tangro/actions-coverage@1.1.0
     with:
       post-comment: true
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       GITHUB_CONTEXT: ${{ toJson(github) }}
 ```
 
+There is also an option to adapt the minimum coverage percentage limits that need to be achived for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
```
Comment (ankri) [inline README.md]:
```suggestion
There is also an option to adapt the minimum coverage percentage limits that need to be achieved for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
```

**Review by ankri — COMMENTED** [review]
Hi,

thank you very much for you Pull Request. I would have never guessed that anyone would have found our actions (because they are not listed), let alone use them (because they are mainly tied to our use case).

I have suggested one simple fix in your changes to `README.MD`. As soon as it is accepted, we will gladly merge this PR.

Thanks again and cheers

File: README.md (hunk 1 above)
Comment (kl4n4) [reply README.md]: that does make sense - my first approach was to use environment variables but I agree, parameters are a better fit for these settings.
So I changed the implementation accordingly.

**Comment by kl4n4** [comment]
Hi,

you are very welcome and thanks for these great actions!
I found them while searching for simple and easy to use github actions for Jest and all others were unnecessary complex or did not work at all. So thanks for putting them OS! They do work great for our projects as well (besides the fact that some legacy apps need a configurable coverage value😉)

I changed the configuration to parameters and fixed the typo in the README. :)

**Pull request merged**